book/src/configuration.md

```diff
@@ -54,7 +54,7 @@ Hello
 To be noted, if one of the trimmed characters is a newline, then the only
 character remaining will be a newline.
 
-If you want this to be the default behaviour, you can set `whitespace` to
+If you want this to be the default behavior, you can set `whitespace` to
```

Comment:
This is british english (left) vs US english (right). Both are valid.

Reply:
Yes, but IMHO we should stick to one locale.

I guess in a French text you would probably replace "nonante", too, even if it is valid Belgian French; as I would probably replace instances if "Jänner" ("January", Swiss) or "heuer" ("this current year", Austrian). :)

Reply:
Nah I prefer nonante, makes much more sense. Although I'm surprised by how many people know about this compared to how many people living in France know about it. :rofl: